src/Raven.Server/Web/System/StudioDatabasesHandler.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.System;
+
+public class StudioDatabasesHandler : RequestHandler
+{
+    [RavenAction("/studio-tasks/databases", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
This needs to be wired to the studio?

Reply:
Not sure I follow?

Reply:
Is it being used by the studio?

Reply:
No, because I just added it and Marcin, who sits next to me, will soon use it, because we spoke about it :)